app/models/cart.rb

```diff
@@ -2,11 +2,15 @@ class Cart
   attr_reader :data
 
   def initialize(data)
-    @data = data || Hash.new
+    @data = data || { "" => 0 }
```

Comment:
I wanted to have zero displayed in the cart before items are added. Didn't know the best way to do this...